tests/bettertransformer/test_bettertransformer_audio.py

```diff
@@ -141,3 +143,109 @@ def test_raise_train(self):
                 bt_model = BetterTransformer.transform(hf_random_model)
                 bt_model.train()
                 _ = bt_model(**inputs)
+
+    @parameterized.expand([(False,)])
+    def test_invert_modules(self, keep_original_model=False):
+        r"""
+        Test that the inverse converted model and hf model have the same modules
+        Since `Wav2vec2` does not support `deepcopy` we cannot test the transformation
+        with `keep_original_model=True` for this model.
+        """
+        for model in self.all_models_to_test:
```

Comment:
Can you rather use `@parameterized.expand`? It allows to run tests individually which is great for debugging.

Reply:
It's a personal taste but I perfer to not rely on `parameterized.expand` for expanding on each `model_id` since it will slowdown the entire testing suite (you will call `setUp` and `tearDown` internally as many times as the number of models will delete & re-create again the intermediate variables. I didn't benchmarked it though so if it'll lead to the same speed happy to change !

Reply:
Fixed in [c5179e3](https://github.com/huggingface/optimum/pull/676/commits/c5179e396f023461fd7b11227d8d6121c0f7c2a7)